docs/troubleshooting_and_faq/docker.md

```diff
@@ -0,0 +1,13 @@
+# Docker
+
+
+### When starting Docker, I get 'Permission denied while trying to connect to the Docker daemon'
+
+If you're having trouble getting Docker to run on your system, you can refer to the Docker documentation for instructions
+on how to start Docker on [macOS](https://docs.docker.com/docker-for-mac/install/), [Linux](https://docs.docker.com/desktop/install/debian/#launch-docker-desktop), and [Windows 10 and 11](https://docs.docker.com/desktop/install/windows-install/#start-docker-desktop).
+
```

Comment:
Didn't we also have something about the socket checkbox in the advanced tab needing to be set? Or if not, should we?